### PR TITLE
[DJ][FEAT-29]: Added endpoint for Getting Categories, Unit Tests and Swagger docs

### PR DESCRIPTION
**Pull request by purple-mountain**

### CODE REVIEW

File: src/database/data/accommodation-categories.data.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { AccommodationCategory } from 'src/modules/accommodations/entities';
+import { AccommodationCategory } from 'src/modules/categories/entities/category.entity';
```
Comment (bek-shoyatbek) [inline src/database/data/accommodation-categories.data.ts:1]: Could you create index file in entities for better import/export?

File: src/modules/accommodations/entities/accommodation.entity.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ export class Accommodation {
 
   @ManyToMany(() => Amenity)
   @JoinTable({
-    name: 'accommodation_amenities', // name of the junction table
```
Comment (bek-shoyatbek) [inline src/modules/accommodations/entities/accommodation.entity.ts:48]: thanks....

**Review by bek-shoyatbek — CHANGES_REQUESTED** [review]

File: src/database/data/accommodation-categories.data.ts (hunk 1 above)
Comment (purple-mountain) [reply src/database/data/accommodation-categories.data.ts]: done

File: src/modules/categories/dto/response/category.dto.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AccommodationCategoryDto {
+  @ApiProperty()
+  id: string;
+
+  @ApiProperty()
+  name: string;
+
+  @ApiProperty()
```
[the remark this answers is not in the document]
Comment (purple-mountain) [reply src/modules/categories/dto/response/category.dto.ts:10]: I'm just returning the data from the db, and the Category entity that you created included createdAt and updatedAt. If these fields aren’t needed, I can make a migration to remove them.

**Review by bek-shoyatbek — APPROVED** [review]

**Pull request merged**